lib/openssl/asn1.rb

```diff
@@ -10,6 +10,41 @@
 
 module OpenSSL
   module ASN1
+    V_ASN1_UNIVERSAL= 0x00
+    V_ASN1_APPLICATION = 0x40
+    V_ASN1_CONTEXT_SPECIFIC = 0x80
+    V_ASN1_PRIVATE = 0xc0
+    V_ASN1_CONSTRUCTED = 0x20
+    V_ASN1_PRIMITIVE_TAG = 0x1f
+
+    V_ASN1_INTEGER            = 2
+    V_ASN1_BIT_STRING         = 3
+    V_ASN1_OCTET_STRING       = 4
+    V_ASN1_NULL               = 5
+    V_ASN1_OBJECT             = 6
+    V_ASN1_OBJECT_DESCRIPTOR  = 7
+    V_ASN1_EXTERNAL           = 8
+    V_ASN1_REAL               = 9
+    V_ASN1_ENUMERATED         = 10
+    V_ASN1_UTF8STRING         = 12
+    V_ASN1_SEQUENCE           = 16
+    V_ASN1_SET                = 17
+    V_ASN1_NUMERICSTRING      = 18
+    V_ASN1_PRINTABLESTRING    = 19
+    V_ASN1_T61STRING          = 20
+    V_ASN1_TELETEXSTRING      = 20
+    V_ASN1_VIDEOTEXSTRING     = 21
+    V_ASN1_IA5STRING          = 22
+    V_ASN1_UTCTIME            = 23
+    V_ASN1_GENERALIZEDTIME    = 24
+    V_ASN1_GRAPHICSTRING      = 25
+    V_ASN1_ISO64STRING        = 26
+    V_ASN1_VISIBLESTRING      = 26
+    V_ASN1_GENERALSTRING      = 27
+    V_ASN1_UNIVERSALSTRING    = 28
+    V_ASN1_BMPSTRING          = 30
```

Comment:
These are duplicate of those defined in `ossl_asn1.c` (but without leading `V_ASN_` in the name)